app/src/main/res/color/color_bg_item.xml

```diff
@@ -17,6 +17,6 @@
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
     <!-- Ideally I would've used "?colorPrimary" like elsewhere but it fails pre API 24.
          The result will be exactly the same though. -->
-    <item android:alpha="0.2" android:color="@color/color_primary" android:state_activated="true"/>
+    <item android:alpha="0.2" android:color="?colorPrimary" android:state_activated="true"/>
```

Comment:
Still fails on API <24, you get the characteristic red error color:
![image](https://user-images.githubusercontent.com/33690959/209449457-cd770a57-231c-465c-a5f5-a6b9fe2a5b5c.png)

If I use `@color/md_theme_light_primary` do you think that's fine? I could also make a separate XML file for API <24 and keep `?colorPrimary` otherwise.

Reply:
I think it would be the better choice to use `?colorPrimary` whenever possible. `?colorPrimary` is either equal to `@color/md_theme_light_primary` or to `@color/md_theme_dark_primary` depending on the app theme.

By always setting the background color to `@color/md_theme_light_primary`, the color would be incorrect in dark mode, I think.

Are you sure that API Level 24 is the correct cutoff? It seemed to work fine on API Level 23 in my testing.

Reply:
I think the cutoff is actually API 23. I added the correct selectors for color, color-night, and color-v23 (with attributes), it should work.